anesthetic/gui/plot.py

```diff
@@ -90,23 +90,27 @@ def set_text(self, logL, n):
 class Temperature(Slider):
     """Logarithmic slider controlling temperature of the posterior points."""
 
-    def __init__(self, fig, gridspec, action):
-        super().__init__(fig, gridspec, action, r'$kT$', -1, 5, 0, 'vertical')
+    def __init__(self, beta, D_KL, fig, gridspec, action):
+        self._beta = beta
+        self._D_KL = D_KL
+        D_KL0 = np.interp(1, beta, D_KL)
+        super().__init__(fig, gridspec, action, r'$\beta$',
+                         D_KL[0], D_KL[-1], D_KL0, 'vertical')
```

Comment:
Out of curiosity, how does D_KL factor into the beta range/scaling? 
Or is the better question why/how does D_KL govern the peak position on the Higson plot?

Reply:
To quote Ed Jaynes via Skilling, "this is well known to those who know such things". The D_KL is the information, i.e. the size of the typical set, so most of the posterior bulk/non-zero weights will appear once nested sampling has compressed by about this much (and dissappear as we move onward ard upward). [Sklling](https://projecteuclid.org/journals/bayesian-analysis/volume-1/issue-4/Nested-sampling-for-general-Bayesian-computation/10.1214/06-BA127.full) covers this throughout the initial nested sampling paper, but it first appears in section 4.